src/universal/components/MeetingDashAlert/MeetingDashAlert.js

```diff
@@ -1,33 +1,54 @@
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, {Component} from 'react';
 import withStyles from 'universal/styles/withStyles';
 import {css} from 'aphrodite-local-styles/no-important';
 import {Link} from 'react-router-dom';
 import plural from 'universal/utils/plural';
 import {DashAlert} from 'universal/components/Dashboard';
+import {commitLocalUpdate} from 'react-relay';
+import withAtmosphere from 'universal/decorators/withAtmosphere/withAtmosphere';
 
-const MeetingDashAlert = (props) => {
-  const {activeMeetings, styles} = props;
-  return (
-    <DashAlert colorPalette="warm">
-      <div className={css(styles.inlineBlock, styles.message)}>
-        {`${plural(activeMeetings.length, 'Meeting')} in progress:`}
-      </div>
-      <div className={css(styles.inlineBlock)}>
-        {activeMeetings.map((meeting) => {
-          return (
-            <Link key={meeting.link} className={css(styles.link)} title="Join Active Meeting" to={meeting.link}>
-              {meeting.name}
-            </Link>
-          );
-        })}
-      </div>
-    </DashAlert>
-  );
-};
+class MeetingDashAlert extends Component {
+  componentDidMount() {
+    const {atmosphere} = this.props;
+    const {viewerId} = atmosphere;
+    commitLocalUpdate(atmosphere, (store) => {
+      store.get(viewerId).setValue(true, 'hasMeetingAlert');
```

Comment:
this replaces the dash duck, which allows us to get rid of that reducer as well as the DashLayoutContainer